test/extended/openstack/cpms.go

```diff
@@ -109,17 +126,30 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] ControlPlane Mach
 				subnetName := subnet.Get("filter.name").String()
 				machineSubnets[subnetName] = struct{}{}
 			}
-			if machineNovaAvailabilityZone != "" {
-				machineNovaAvailabilityZones = append(machineNovaAvailabilityZones, machineNovaAvailabilityZone)
+			if machineNovaAz != "" || machineCinderAz != "" || machineVolumeType != "" {
+				machinesFDs = append(machinesFDs, FailureDomain{
+					ComputeZone: machineNovaAz,
+					VolumeZone:  machineCinderAz,
+					VolumeType:  machineVolumeType,
+				})
 			}
+
+			for _, network := range objects(machine.Get("spec.providerSpec.value.networks")) {
+				subnet := network.Get("subnets").String()
+
+				if subnet == "" {
+					machineNetworks = append(machineNetworks, network["uuid"].(string))
+				}
+			}
+
 			o.Expect(machineFlavor).To(o.Equal(cpmsFlavor), "flavor mismatch on Machine %q", machineName)
 			o.Expect(machineImage).To(o.Equal(cpmsImage), "image mismatch on Machine %q", machineName)
 			o.Expect(machineSecurityGroups).To(o.Equal(cpmsSecurityGroups), "security group mismatch on Machine %q", machineName)
 			o.Expect(machineSubnets).To(o.Equal(cpmsSubnets), "subnets mismatch on Machine %q", machineName)
 			o.Expect(machineServerGroup).To(o.Equal(cpmsServerGroup), "server group	 mismatch on Machine %q", machineName)
+			o.Expect(machineNetworks).To(o.Equal(cpmsNetworks), "Network mismatch on Machine %q", machineName)
```

Comment:
Do you think it makes sense to use `o.ConsistOf` here as well?

Reply:
We care about the order here , right? And o.consistOf doesn't care about it (right?)

Reply:
I propose ConsistOf specifically because it doesn't care about order, and I mistakenly thought we also didn't.